tests/build_/test_arrangements.py

```diff
@@ -181,1213 +181,83 @@ def get_pulp_sync_registry(self, conf):
-        plugins = get_plugins_from_build_json(build_json)
-        args = plugin_value_get(plugins, phase, plugin, 'args')
-
-        docker_registry = self.get_pulp_sync_registry(osbs.build_conf)
-        assert args == {'registries': {docker_registry: {'insecure': True}}}
-
-    @pytest.mark.parametrize('osbs, with_group', [  # noqa:F811
-        ({'platform_descriptors': {'x86_64': {'architecture': 'amd64'}},
-            'additional_config': get_pulp_additional_config(),
-            'kwargs': {'registry_uri': 'registry.example.com/v2'}}, False),
-        ({'platform_descriptors': {'x86_64': {'architecture': 'amd64'}},
-            'additional_config': get_pulp_additional_config(with_group=True),
-            'kwargs': {'registry_uri': 'registry.example.com/v2'}}, True)],
-        indirect=['osbs'])
-    def test_group_manifests(self, osbs, with_group):
-        additional_params = {
-            'base_image': 'fedora:latest',
-        }
-        _, build_json = self.get_orchestrator_build_request(osbs, additional_params)
-        plugins = get_plugins_from_build_json(build_json)
-
-        args = plugin_value_get(plugins, 'postbuild_plugins', PLUGIN_GROUP_MANIFESTS_KEY, 'args')
-        docker_registry = self.get_pulp_sync_registry(osbs.build_conf)
-
-        expected_args = {
-            'goarch': {'x86_64': 'amd64'},
-            'group': with_group,
-            'registries': {docker_registry: {'insecure': True, 'version': 'v2'}}
-        }
-        assert args == expected_args
-
-    @pytest.mark.parametrize('build_type', (  # noqa:F811
-        'orchestrator',
-        'worker',
-    ))
-    def test_inject_parent_image(self, osbs, build_type):
-        additional_params = {
-            'base_image': 'foo',
-            'koji_parent_build': 'fedora-26-9',
-        }
-        _, build_json = self.get_build_request(build_type, osbs, additional_params)
-        plugins = get_plugins_from_build_json(build_json)
-
-        args = plugin_value_get(plugins, 'prebuild_plugins', 'inject_parent_image', 'args')
-        expected_args = {
-            'koji_parent_build': 'fedora-26-9',
-            'koji_hub': osbs.build_conf.get_kojihub()
-        }
-        assert args == expected_args
-
-    @pytest.mark.parametrize('worker', [False, True])  # noqa:F811
-    @pytest.mark.parametrize('scratch', [False, True])
-    def test_flatpak(self, osbs, worker, scratch):
-        additional_params = {
-            'flatpak': True,
-            'target': 'koji-target',
-        }
-        if scratch:
-            additional_params['scratch'] = True
-        if worker:
-            additional_params['compose_ids'] = [42]
-            params, build_json = self.get_worker_build_request(osbs, additional_params)
-        else:
-            params, build_json = self.get_orchestrator_build_request(osbs, additional_params)
-        plugins = get_plugins_from_build_json(build_json)
+class UnsupportedArrangement(object):
```

Comment:
I'm curious if  is it worth to keep such testing. It has nice history overview of how arrangements version evolved but in reality it tests only validate_arrangement_version in a complicated way.

Reply:
Yeah, I kept those test classes mostly just for the documentation value, but if you don't want them they can go.

Reply:
historical documentation value can be retrieved from git, so get rid of them please.